### PR TITLE
Resolve IDENT-3524 Fixed multitenant protocol session opening

### PR DESCRIPTION
**Pull request by dejsenlitro**

### CODE REVIEW

File: aries_cloudagent/multitenant/admin/tests/test_routes.py [hunk 1]
```diff
@@ -13,15 +13,19 @@
...
             __aexit__=async_mock.CoroutineMock(), autospec=True
         )
-        mock_multitenant_mgr.__aenter__ = async_mock.CoroutineMock(
-            return_value=mock_multitenant_mgr
+        self.mock_multitenant_mgr.__aenter__ = async_mock.CoroutineMock(
+            return_value=self.mock_multitenant_mgr
         )
 
-        self.session_inject = {BaseMultitenantManager: mock_multitenant_mgr}
+        self.session_inject = {BaseMultitenantManager: self.mock_multitenant_mgr}
```
Comment (acuderman) [inline aries_cloudagent/multitenant/admin/tests/test_routes.py:23]: can we get rid of this?

File: aries_cloudagent/multitenant/admin/routes.py [hunk 2]
```diff
@@ -400,22 +401,22 @@ async def wallet_create_token(request: web.BaseRequest):
         body = await request.json()
         wallet_key = body.get("wallet_key")
 
-    async with context.session() as session:
-        try:
-            multitenant_mgr = session.inject(BaseMultitenantManager)
+    try:
+        multitenant_mgr = context.profile.inject(BaseMultitenantManager)
+        async with context.session() as session:
```
Comment (vidp1) [inline aries_cloudagent/multitenant/admin/routes.py:406]: profile.session() pls.

File: aries_cloudagent/multitenant/admin/routes.py [hunk 3]
```diff
@@ -444,22 +445,22 @@ async def wallet_remove(request: web.BaseRequest):
...
```
Comment (vidp1) [inline aries_cloudagent/multitenant/admin/routes.py:450]: Same as above.

File: aries_cloudagent/multitenant/admin/tests/test_routes.py [hunk 4]
```diff
@@ -13,15 +13,19 @@
...
         self.context = AdminRequestContext.test_context(self.session_inject)
+        self.context.profile.context.injector.bind_instance(
```
[the remark this answers is not in the document]
Comment (dejsenlitro) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py:25]: Tried. Doesn't work

File: aries_cloudagent/multitenant/admin/routes.py (hunk 3 above)
Comment (dejsenlitro) [reply aries_cloudagent/multitenant/admin/routes.py:452]: Fixed

**Review by vidp1 — APPROVED** [review]

**Pull request merged**